test/2d/test_2d.cpp

```diff
@@ -241,13 +241,13 @@ TEST(Edge, canny)
   edge_->setHysteresisThresholdHigh (50);
   edge_->detectEdgeCanny (*output_cloud);
 
-  int count = 0;
+  //int count = 0;
```

Comment:
I'd say let's remove this entirely, including the commented-out `EXPECT_LE` below.